recipes/dftatom/bld.bat

```diff
@@ -0,0 +1,17 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -D CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE ^
```

Comment:
@certik you might be interested in including this line in your CMake. I think it also limits the CMake backwards compatibility though.

Reply:
I created an issue for that: https://github.com/certik/dftatom/issues/27.